src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -130,8 +134,11 @@ export default {
 		categories() {
 			// merge realCategories & customCategories
 			const categories = _uniqBy(this.realCategories.concat(this.customCategories, this.clientCategories), 'id');
-			// fiter our any empty categories and re-order to match the setting
-			return categories.filter(channel => channel.loans !== null).sort(indexIn(this.categoryIds, 'id'));
+			// fiter our any empty categories and categories with 0 loans
+			return categories.filter(channel => {
+				return channel.loans !== null && channel.loans.values.length;
```

Comment:
@emuvente added a check to ensure we have loans before allowing a category into the array. Water and Sanitation got us again... Also verified that this did not affect page tracking data.